tests/Temporalio.Tests/TestUtils.cs

```diff
@@ -224,4 +225,60 @@ public record CaptureMetric<T>(string Name, string? Unit, string? Description)
 
         public void Set(T value, object tags) => Values.Enqueue((value, (Dictionary<string, object>)tags));
     }
-}
\ No newline at end of file
+
+    public static async Task<Temporalio.Api.WorkflowService.V1.SetWorkerDeploymentRampingVersionResponse>
```

Comment:
Arguably this utility and that last one doesn't provide much value vs inlining the call heh